DeviceDetector.php

```diff
@@ -1107,6 +1107,7 @@ protected function parseDevice(): void
         if (\in_array($clientName, [
             'Kylo', 'Espial TV Browser', 'LUJO TV Browser', 'LogicUI TV Browser', 'Open TV Browser', 'Seraphic Sraf',
             'Opera Devices', 'Crow Browser', 'Vewd Browser', 'TiviMate', 'Quick Search TV', 'QJY TV Browser', 'TV Bro',
+            'Redline',
```

Comment:
Might be good to also add a full useragent test, that ensures the detection as `TV` is covered.